hooks/post_playblast.py

```diff
@@ -16,98 +16,85 @@
 import pymel.core as pm
 import pprint
 
-import tank
-from tank import Hook
+import sgtk
 
-from datetime import datetime 
+from datetime import datetime
 TIMESTAMP_FORMAT = "%Y-%m-%d %H:%M:%S"
 
-class PostPlayblast(Hook):
+class PostPlayblast(sgtk.Hook):
```

Comment:
Try switching this to use the newer hook base class method
```suggestion
HookClass = sgtk.get_hook_baseclass()

class PostPlayblast(HookClass):
```